presto-spi/src/main/java/com/facebook/presto/spi/block/AbstractVariableWidthBlock.java

```diff
@@ -25,7 +25,8 @@
 {
     protected abstract Slice getRawSlice(int position);
 
-    protected abstract int getPositionOffset(int position);
+    // TODO: expose offset in ColumnarSlice
```

Comment:
Is this still a TODO?

Reply:
Yes. Currently I exposed it in AbstractVariableWidthBlock. If we decide to write a ColumnarSlice similar to ColumnarRow/Array/Map, then we want to expose the offset in  ColumnarSlice instead of AbstractVariableWidthBlock

Reply:
nit:
```
// Consider create `ColumnarSlice` class and expose the offset through it
```

The first impression when I read the comment I thought we already have `ColumnarSlice` class.